src/lib/gundb/UserStorageClass.js

```diff
@@ -1086,7 +1086,18 @@ export class UserStorage {
     this.addStartClaimingCard()
 
     if (Config.enableInvites) {
-      setTimeout(() => this.enqueueTX(inviteFriendsMessage), 120000) // 2 minutes
+      inviteFriendsMessage.id = '0.1'
```

Comment:
please define that values (.1, .2) as constants to avoid issues if we would need to change them

Reply:
done